### PR TITLE
install libssl1.0.0 manually if needed, use https

### PR DESCRIPTION
**Pull request by bryceml**

### CODE REVIEW

**Comment by brandleesee** [comment]
Have merged in hopes that this facilitates the issue in https://github.com/brandleesee/ePSXe64Ubuntu/issues/11

File: ePSXe64Ubuntu.sh [hunk 1]
```diff
@@ -25,11 +25,23 @@ tput setaf 1; echo "  Add to Favorites/Lock"; tput sgr0
 tput setaf 1; echo "  CLOSE ePSXe GUI to continue with the script."; tput sgr0
 tput setaf 2; echo "Script started."; tput sgr0
 
+sudo apt-get update
+sudo apt-get -y install wget
```
Comment (brandleesee) [inline ePSXe64Ubuntu.sh:29]: @bryceml want to ask, how come including to install wget when script is initiated using wget?
Could it be that latest patch requires latest version of wget?
Cheers, mate!

```
wget -O ePSXe64Ubuntu.sh https://raw.githubusercontent.com/brandleesee/ePSXe64Ubuntu/master/ePSXe64Ubuntu.sh && bash ePSXe64Ubuntu.sh
```

Comment (bryceml) [reply ePSXe64Ubuntu.sh:29]: I don't always fetch the script via wget, sometimes I git clone it.  I was making both easier on myself and more foolproof for others by making sure it was installed for the rest of the script.

Comment (brandleesee) [reply ePSXe64Ubuntu.sh:29]: I see. In the same spirit, should we add other packages to fetch besides wget?

**Pull request merged**